.rubocop.yml

```diff
@@ -0,0 +1,11 @@
+AllCops:
+  Exclude:
+    - 'db/**/*'
+    - 'config/**/*'
+    - 'script/**/*'
+    - 'bin/{rails,rake,setup,update}'
+    - !ruby/regexp /old_and_unused\.rb$/
```

Comment:
This looks awesome!   Way to make sure files that don't need testing get ignored!